src/Commands/SystemCommands/GenericmessageCommand.php

```diff
@@ -68,9 +68,57 @@ public function execute()
 
         //Fetch conversation command if it exists and execute it
         if ($conversation->exists() && ($command = $conversation->getCommand())) {
-            return $this->telegram->executeCommand($command);
+            return $this->getTelegram()->executeCommand($command);
+        }
+
+        // Try to execute any deprecated system commands.
+        if ($deprecated_system_command_response = $this->executeDeprecatedSystemCommand()) {
```

Comment:
I think `executeNoDb()` should call this as well?

Reply:
Absolutely!